disttask/framework/framework_dynamic_dispatch_test.go

```diff
@@ -94,6 +91,7 @@ func TestFrameworkDynamicBasic(t *testing.T) {
 }
 
 func TestFrameworkDynamicHA(t *testing.T) {
+	t.Skip("skip this test because the function is not implemented yet")
```

Comment:
Why skip it?

Reply:
reverted https://github.com/pingcap/tidb/pull/46957/commits/e573f2d2a9328371369593386ba119c764b2e281